### PR TITLE
[office-js, office-js-preview] Add note about the Binding ID that's returned by binding events

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: 
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -20384,7 +20384,8 @@ declare namespace Excel {
     interface BindingSelectionChangedEventArgs {
         /**
          *
-         * Gets the Binding object that represents the binding that raised the SelectionChanged event.
+         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:20387]: The "Gets ..." terminology makes it sound like this is a description of a method or property, but there's no method or property signature below the description.

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (AlexJerabek) [reply types/office-js-preview/index.d.ts:20387]: @Rick-Kirkham There is a property there:

```
    interface BindingSelectionChangedEventArgs {
        /**
         *
         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. 
         * Use that ID with `BindingCollection.getItem(id)` to get the binding.
         *
         * [Api set: ExcelApi 1.2]
         */
        binding: Excel.Binding;

....

        /**
         *
         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. 
         * Use that ID with `BindingCollection.getItem(id)` to get the binding.
         *
         * [Api set: ExcelApi 1.2]
         */
        binding: Excel.Binding;

```

Comment (Rick-Kirkham) [reply types/office-js-preview/index.d.ts:20387]: Strange. It doesn't appear for me in the diff for either of the events in both files. Here's a screenshot of the one just below this one:
image

**Review by Rick-Kirkham — APPROVED** [review]

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (AlexJerabek) [reply types/office-js-preview/index.d.ts:20387]: I was having a similar issue earlier. Maybe it's a recent GitHub bug?

**Pull request merged**